images/runtime/python/cgmanifest.json

```diff
@@ -4,7 +4,7 @@
             "Component": {
                 "pip": {
                     "Name": "viztracer",
-                    "Version": "0.14.3"
+                    "Version": "0.15.5"
```

Comment:
Was this a manual update? We may need to make another entry in this file if we support both `viztracer@0.14.3` and `viztracer@0.15.5`

Reply:
Since we are installing viztracer version 0.15.5 across all python versions, do we still need  this?

Reply:
Since we're installing `0.15.5` across the board, we should be good to keep this change as-is 👍